### PR TITLE
Upgrade `Pagination` comp. + fix `ButtonV2` + adj. vertical-align of `CareIcons` + Generalize `Submit` and `Cancel` buttons

### LINKED ISSUE #4343: Misalignment across the platform across multiple location
**Describe the bug**

The icon and text inside the  buttons are misaligned in the:

- [ ] bed capacity
- [ ] Add location page
- [ ] Add bed page

*Steps to Reproduce*

- https://care.coronasafe.in/facility/52622c36-0ebf-4699-a9d2-3d4a18cfd40a/bed
- https://care.coronasafe.in/facility/52622c36-0ebf-4699-a9d2-3d4a18cfd40a/location/add
- https://care.coronasafe.in/facility/52622c36-0ebf-4699-a9d2-3d4a18cfd40a/location/d9374365-1838-4163-84d7-e3a3c1e85154/beds/add

*Screenshots*

image
image
image

### LINKED ISSUE #4347: Misalignment across the platform across asset page
**Describe the bug**

The icon and text inside the buttons are misaligned in the:

- [ ] configure asset page ( Both vital and camera page )
- [ ] asset info page

**To Reproduce**

- https://care.coronasafe.in/facility/0fd9dfa1-e4fb-4bbb-936d-c3cb05fa5e33/assets/a2bc8f07-d485-46ed-bcae-a850797da04c/configure
- https://care.coronasafe.in/facility/52622c36-0ebf-4699-a9d2-3d4a18cfd40a/assets/0b4f03af-c60c-475b-8fe5-e162331a9354

**Screenshots**

image
image

### LINKED ISSUE #4350: Use angle icons instead of `<<<`, `<`, `>>>`, `>` texts in pagination component.
**Describe the bug**
Use angle icons instead of `<<<`, `<`, `>>>`, `>` texts in the pagination component.

**To Reproduce**

1. Go to any page with pagination

**Expected behavior**
Use angle icons for pagination component.

**Screenshots**

image

image

### LINKED ISSUE #4367: Goodbye `CheckCircleOutline` icon of material UI
**Describe the bug**
`CheckCircleOutlineIcon` is the most popularly used Material UI icon in the codebase. In effort to push out material UI, it shall be migrated to CareIcon's `check` / `check-circle` equivalent.

**Expected behavior**
Replace with any one of the following. (not both, as we need to be consistent with our design)
- `<CareIcon className="care-l-check text-lg" />`
- `<CareIcon className="care-l-check-circle text-lg" />`

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Fixes #4347
- Fixes #4343
- Resolves #4350 
- Fixes the misalignment in `ButtonV2` caused by `Button` class getting deleted from the codebase.
- Use `CareIcons` for the angles of the pagination component
- Use `ButtonV2` for navigation buttons of the pagination component
- Added tooltips for navigation buttons of the pagination component
- Use `text-*` classes to size icons instead of `h-*`
- Reduce the `.care-svg-icon__baseline`'s vertical alignment to -0.175em
- Resolves #4367
- Replaces `CheckCircleOutline` and it's containing buttons to `CareIcon` and `ButtonV2` in favour of removing material UI components.

### Note
From now on we need not adjust the alignment of the icon inside button manually using `pt-*` / `pb-*` / `mt-*` / `mb-*` classes.

```tsx
<ButtonV2> {/* already applies "inline-flex items-center gap-2" */}
  <CareIcon className="care-l-check text-lg" />
  <span>Verify</span>
</ButtonV2>
```

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

image

image

image

image

image

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [x] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by Ashesh3 — APPROVED** [review]
LGTM

**Comment by Ashesh3** [comment]
@rithviknishad The Cypress tests might need to be updated too.

image

**Comment by rithviknishad** [comment]
@Ashesh3 yea on it. just finished, pushing it

**Comment by nihal467** [comment]
@rithviknishad 
image

does the circle around the tick was intentionally removed in your PR

**Comment by rithviknishad** [comment]
@nihal467 yes.

**Comment by nihal467** [comment]
@rithviknishad 
image

the right side pagination button are missing in facility page

**Comment by rithviknishad** [comment]
@nihal467 fixed. could you check now

**Comment by nihal467** [comment]
@rithviknishad 
image

when you are removing the circle only in some places, we will be losing consistency, since there are a lot of places we still use the tick inside the circle icon. What do you think?

**Comment by rithviknishad** [comment]
Merged latest develop too, cypress would pass now

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
@rithviknishad 
image

https://deploy-preview-4358--care-net.netlify.app/facility/e3311351-3ec9-4cb0-a6e1-902225ac705c/assets/175b65f0-e7df-4487-a1bf-2fab7c8c6a63/update

the asset page, the cancel button should be having a cross, which is similar to the existing staging

image

https://deploy-preview-4358--care-net.netlify.app/facility/e3311351-3ec9-4cb0-a6e1-902225ac705c/location/1a2eea2a-5e51-41fb-a7ab-012d04959f1d/beds/add

the cancel button is the bed capacity is missing the circle around the cross

image

https://deploy-preview-4358--care-net.netlify.app/facility/e3311351-3ec9-4cb0-a6e1-902225ac705c/inventory/add

the inventory cancel button is malfunctioning, as it is saying the inventory was created successfully. ( Existing staging not having this problem )

**Review by Ashesh3 — APPROVED** [review]
LGTM. Finally standardized Cancel and Submit buttons 🎉

File: src/Components/Common/components/ButtonV2.tsx [hunk 1]
```diff
@@ -96,3 +97,42 @@ const ButtonV2 = ({
 };
 
 export default ButtonV2;
+
+// Common buttons
+
+type CommonButtonProps = ButtonProps & { label?: string; icon?: string };
```
Comment (rithviknishad) [inline src/Components/Common/components/ButtonV2.tsx]:
```suggestion
type CommonButtonProps = ButtonProps & { label?: string; };
```

**Comment by nihal467** [comment]
@rithviknishad 
image

make the proper spacing between the buttons

File: src/Components/Facility/DoctorCapacityForm.tsx [hunk 2]
```diff
@@ -244,63 +243,44 @@ export const DoctorCapacityForm = (props: DoctorCapacityProps) => {
...
-                    variant="secondary"
-                    className="w-full sm:w-auto"
+                  <Cancel
                     onClick={() => goBack(!id && `/facility/${facilityId}`)}
-                  >
-                    Cancel
-                  </ButtonV2>
+                  />
                 </div>
                 <div className="flex flex-row w-full sm:w-auto flex-wrap">
```
Comment (rithviknishad) [inline src/Components/Facility/DoctorCapacityForm.tsx]:
```suggestion
                <div className="flex flex-row w-full sm:w-auto flex-wrap gap-2">
```

**Comment by nihal467** [comment]
@rithviknishad 
image

keep the add present towards the right side, as of the existing staging

File: src/Components/Assets/configure/CameraConfigure.tsx [hunk 3]
```diff
@@ -63,15 +63,7 @@ export default function CameraConfigure(props: CameraConfigureProps) {
               </div>
             </div>
             <div className="flex justify-between mt-4">
```
Comment (rithviknishad) [inline src/Components/Assets/configure/CameraConfigure.tsx]:
```suggestion
            <div className="flex justify-end mt-4">
```

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@rithviknishad can you checkout the merge conflict

**Pull request merged**